src/plasmapy/formulary/collisions/misc.py

```diff
@@ -436,6 +437,40 @@ def scattering_integrand(theta, B):
 
     return scattering_integrand
 
+def Highland_scattering(
+    m: u.Quantity[u.kg],
+    v: u.Quantity[u.m / u.s],
+    L_rad: u.Quantity[u.kg / u.m**2],
+    dt: u.Quantity[u.s],
+):
+    """Calculate the RMS scattering angle using the Highland formula.
+
+    Parameters
+    ----------
+    m : `~astropy.Units.Quantity`
+        The mass of the projectile particles streaming through the target.
+    v : `~astropy.units.Quantity`
+        The speed of the projectile particles streaming through the target.
+    L_rad : `~astropy.units.Quantity`
+        The radiation length of the target material. See notes for more details.
+    dt : `~astropy.units.Quantity`
+        The timestep being used in the Monte Carlo simulation.
+
+    Returns
+    -------
+    theta_rms : `~astropy.units.Quantity`
+        The RMS scattering angle in radians.
+    """
```

Comment:
Add a reference to the Highland paper in the docstring, and maybe a few notes on the regimes of relevance as described in the paper (e.g. E_s may not be constant below Z=20)

Reply:
I just pushed a draft of the notes section for the `Highland_scattering` function in the collision formulary. I'm still not 100% on the details of the radiation length, but it seems like it's related to energy loss due to bremsstrahlung? I just wrote in the doc string that it was the relevant distance scale when we're talking about these "charge interactions"

Reply:
I think it is used for protons/ions, note that the 'projectile mass' is refered to rather than just the electron mass